ci/create_codewind_index.py

```diff
@@ -4,21 +4,23 @@
 import json
 import os
 import fnmatch
-import sys
 from collections import OrderedDict
+import argparse
+from argparse import ArgumentDefaultsHelpFormatter
 
-base_dir = os.path.abspath(os.path.dirname(sys.argv[0]))
+parser = argparse.ArgumentParser(formatter_class=ArgumentDefaultsHelpFormatter)
 
-displayNamePrefix = "Appsody"
-if len(sys.argv) > 1:
-    displayNamePrefix = sys.argv[1]
+parser.add_argument("-n", "--namePrefix", help="Display name prefix.", default="Appsody")
+parser.add_argument("-f", "--file", help="Location of yaml files.", default=os.getcwd())
```

Comment:
Perhaps we need to make this clear if this is relative or absolute path.. Also is this path to a drectory or to a yaml file

Reply:
It can be the relative or absolute path to either a file or directory